src/components/checklist/Checklist.jsx

```diff
@@ -0,0 +1,80 @@
+/** 
+ * This component is a styled component and contains a Checklist.
+ * 
+ * The Checklist contains of a H2 heading, a list of tasks, an input field for adding new tasks, a delete button and a add task button. 
+ * 
+ * To add: Mark all and task counter.
+ */
+
+import React from 'react';
```

Comment:
No need to import React

Reply:
I started with not importing it, but got a lot of errors in a couple of files, all saying something about react. So, my two cents of a workaround, to quickly fix this before my demo, was to add that import line to the files where the errors occurred, because somehow it makes the app work. 

I will look into this as soon as I can.